code/modules/admin/admin.dm

```diff
@@ -733,6 +733,63 @@ GLOBAL_VAR_INIT(nologevent, 0)
 	log_admin("[key_name(usr)] spawned [chosen] at ([usr.x],[usr.y],[usr.z])")
 	SSblackbox.record_feedback("tally", "admin_verb", 1, "Spawn Atom") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/datum/admins/proc/create_crate(object as text)
```

Comment:
For the love of god do not scope this here, scope it on /client please

Reply:
apologies, spawn_atom seems to be a relic of the past. I'll move this new proc to the other client admin proc file for consistency